Classes/Flow2Lab/EmberAdapter/Configuration/Source/YamlModelConfigurationSource.php

```diff
@@ -65,11 +64,17 @@ public function isClassEmberModel($className) {
 	/**
 	 * @param object $object
 	 * @return string Ember model name
-	 * @throws ConfigurationNotAvailableException
+	 * @throws \InvalidArgumentException If the given object is not annotated as ember model
 	 */
 	public function getModelNameByObject($object) {
-		// todo: implement
-		throw new ConfigurationNotAvailableException();
+		$className = $this->reflectionService->getClassNameByObject($object);
+		if ($this->isClassEmberModel($className) === FALSE) {
```

Comment:
throw ConfigurationNotAvailableException if

```
'MyModel':
+  className: MyPackage\Key\Domain\Model\MyModel
```

Is not set for the given object class